app/javascript/packages/document-capture/context/in-person.ts

```diff
@@ -15,6 +15,11 @@ export interface InPersonContextProps {
    * Date communicated to users regarding expected update about their enrollment after an outage
    */
   inPersonOutageExpectedUpdateDate?: string;
+
+  /**
+   * When true users must enter a full address when searching for a Post Office location
+   */
+  inPersonFullAddressEntryEnabled: boolean;
```

Comment:
This is addressing [Andrew's feedback](https://github.com/18F/identity-idp/pull/8808#discussion_r1268545197) from the prior PR for a cleaner way to pass the feature flag into the Document Capture context